bentoml/saved_bundle/bundler.py

```diff
@@ -93,6 +93,12 @@ def _write_bento_content_to_dir(bento_service, path):
     # write conda environment, requirement.txt
     bento_service.env.infer_pip_packages(bento_service)
     bento_service.env.save(path)
+    # Also write conda environment.yml to module base path to make it accessible when
+    # users specificed custom conda environment.
+    shutil.copyfile(
```

Comment:
why is this fixing the issue?

Reply:
the `bentoml.yml` was duplicated in that directory for a completely different reason, we should not duplicate the env yml file here - see comment below